src/components/SelectInput/index.js

```diff
@@ -11,9 +11,13 @@ const SelectInput = ({ onChange, name, options, value }) => (
   </Select>
 );
 
+SelectInput.defaultValues = {
+  onChange: () => { },
```

Comment:
Didn't the linter warn about the space between the braces?

Reply:
not, strange.